include/seqan3/search/configuration/max_error_rate.hpp

```diff
@@ -38,10 +39,10 @@ namespace seqan3::search_cfg
 template <typename ...errors_t>
 //!\cond
     requires (sizeof...(errors_t) <= 4) &&
-             ((detail::is_type_specialisation_of_v<std::remove_reference_t<errors_t>, total> ||
-               detail::is_type_specialisation_of_v<std::remove_reference_t<errors_t>, substitution> ||
-               detail::is_type_specialisation_of_v<std::remove_reference_t<errors_t>, deletion>  ||
-               detail::is_type_specialisation_of_v<std::remove_reference_t<errors_t>, insertion>) && ...)
+             ((seqan3::detail::is_type_specialisation_of_v<std::remove_reference_t<errors_t>, total> ||
```

Comment:
Can you maybe merge this commit with the one that introduces the error? s.t. the commit history is mostly "stable" meaning that if I do git bisect I have commits that pass CI?

Reply:
I think all commits are compilable and runnable. At least I spent a lot of time to verify this. But I'll check again.

Reply:
Ah but this is only for the header test. Boah, no one is bisecting the header test, right?! That would take hours because you have to compile everything.

Reply:
true, I just read test not header test. leave it as it is.